src/parser/druid/restoration/modules/features/WildGrowth.js

```diff
@@ -68,6 +115,10 @@ class WildGrowth extends Analyzer {
     return (this.belowRecommendedCasts / this.wgs) || 0;
   }
 
+  get percentBelowRecommendedCasts2() {
```

Comment:
I would rename the functions that have '2' in the name to include something more indicative about what the recommendation actually is (in this case it appears to be bad pre-casts or wasted initial WG tick healing). Just a suggestion -- not a requirement.

Reply:
Thanks, was being lazy at the time. :)